addon/bootstrap.js

```diff
@@ -60,18 +59,9 @@ const appStartupObserver = {
   }
 }
 
-const cacheBustReasons = [ADDON_INSTALL, ADDON_UPGRADE, ADDON_DOWNGRADE];
+const APP_STARTUP = 1;
```

Comment:
You can remove this line. Mossop pointed out in the last review that the bootstrap [reason constants](https://developer.mozilla.org/en-US/Add-ons/Bootstrapped_extensions#Reason_constants) are already defined inside bootstrap.js.